daprdocs/content/en/reference/api/cryptography_api.md

```diff
@@ -0,0 +1,128 @@
+---
+type: docs
+title: "Cryptography API reference"
+linkTitle: "Cryptography API"
+description: "Detailed documentation on the cryptography API"
+weight: 1300
+---
+
+Dapr provides cross-platform and cross-language support for encryption and decryption support via the 
```

Comment:
Please add a note that the HTTP APIs are not recommended besides development and testing. Using the SDKs, which use the gRPC APIs, is strongly recommended

Reply:
As this is already merged, added this suggestion to https://github.com/dapr/docs/pull/3967